client/gutenberg/editor/controller.js

```diff
@@ -198,6 +198,11 @@ function getPressThisData( query ) {
 	return url ? { text, url, title, image, embed } : null;
 }
 
+function getAnchorFmData( query ) {
+	const { anchor_podcast, anchor_episode } = query;
+	return anchor_podcast && anchor_episode ? { anchor_podcast, anchor_episode } : null;
```

Comment:
I think the `anchor_episode` is an optional param, which will be only present as part of the Anchor → WP.com flow that prompts Anchor users to transcribe a podcast episode.

The goal of the other flow (WP.com → Anchor → WP.com) is to insert a badge that links to the Spotify show, but this show lists all the episodes, it's not a single-episode purpose link.

Note for example how the Spotify link in https://anchor.fm/bravenotperfect/episodes/Bravery-in-the-Corporate-World-ft--Indra-Nooyi-emif2i/a-a3s5gnl redirects to a list of all episodes.